packages/python-peid/PKGBUILD

```diff
@@ -0,0 +1,34 @@
+# This file is part of BlackArch Linux ( https://www.blackarch.org/ ).
+# See COPYING for license details.
+
+pkgname=python-peid
+_pyname=peid
+pkgver=1.2.9
+pkgrel=1
+pkgdesc='Python implementation of the Packed Executable iDentifier (PEiD)'
+arch=('any')
+grouos=('blackarch' 'blackarch-binary' 'blackarch-reversing')
+license=('GPL3')
+depends=('python' 'python-pefile' 'python-pip')
+makedepends=('python-setuptools')
+source=("$pkgname::git+https://github.com/packing-box/peid")
+sha512sums=('SKIP')
+
+pkgver() {
+  cd $pkgname
+
+  cat src/peid/VERSION.txt
+}
+
+build() {
+  cd $pkgname
+
+  pip wheel .
+}
+
+package() {
```

Comment:
please use a template from https://github.com/BlackArch/blackarch-pkgbuilds, [PKGBUILD-python-PEP517](https://github.com/BlackArch/blackarch-pkgbuilds/blob/master/PKGBUILD-python-PEP517) as there is a [pyproject.toml](https://github.com/packing-box/peid/blob/main/pyproject.toml)